tomviz/templates/Back_Projection_Reconstruction.tvsm

```diff
@@ -0,0 +1,247 @@
+{
+    "operators": [
+        {
+            "argumentTypeInformation": {
+                "threshold": "double"
+            },
+            "arguments": {
+                "threshold": 5
+            },
+            "description": "",
+            "id": "0x56251acc7eb0",
+            "label": "Remove Bad Pixels",
+            "script": "def transform(dataset, threshold=None):\n    \"\"\"Remove bad pixels in tilt series.\"\"\"\n\n    import scipy.ndimage\n    import numpy as np\n\n    tiltSeries = dataset.active_scalars.astype(np.float32)\n\n    for i in range(tiltSeries.shape[2]):\n        I = tiltSeries[:, :, i]\n        I_pad = np.lib.pad(I, (1, 1), 'edge')\n\n        # calculate standard deviation in a 3 x 3 window\n        averageI2 = scipy.ndimage.filters.uniform_filter(I_pad ** 2)\n        averageI = scipy.ndimage.filters.uniform_filter(I_pad)\n        std = np.sqrt(abs(averageI2 - averageI**2))[1:-1, 1:-1]\n\n        medianI = scipy.ndimage.filters.median_filter(I_pad, 2)[1:-1, 1:-1]\n\n        #identiy bad pixels\n        badPixelsMask = abs(I - medianI) > std * threshold\n\n        I[badPixelsMask] = medianI[badPixelsMask]\n        tiltSeries[:, :, i] = I\n\n    # Set the result as the new scalars.\n    dataset.active_scalars = tiltSeries\n",
+            "type": "Python"
+        },
+        {
+            "id": "0x7f3e04014550",
+            "offsets": [
```

Comment:
Will this be OK with image stacks that have different numbers of slices? I wonder what we need to do to make this more of a template, i.e. can we leave settings like this empty to then use the default values?